### PR TITLE
Edited to bring some consistency to the writeup.

### PR DESCRIPTION
**Pull request by nanjekyejoannah-zz**

### CODE REVIEW

File: writeups/Ceph_Multilaguage-RGW-Testing_JoannahNanjekye.md [hunk 1]
```diff
@@ -1,68 +1,235 @@
...
+Ceph is a distributed object, block, and file storage platform. It features an interface for Amazon S3 objects inside of its Rados Gateway (RGW).
 
-One day I gave a talk at a meetup dubbed Geek night kampala on the transition from continous integration to delivery where I met some ladies who requested me to teach them alittle more on the topic a request I welcomed. After getting to know these ladies one of them told me about Rails Girls Summer of Code and suggested that we team up with her and apply for the same. Well we did and got accepted. This was the beginning to a dream coming true. I quit my job just before summer could begin in preparation for the golden opportunity.
+This RGW's S3 interface is tested and currently has three test suites in python, java and Golang that run against it. These test suites use Amazon SDKs namely boto for python, Go sdk for golang and the Java SDK for Java. In future there may be a c++ test suite as well.
```
Comment (nikhita) [inline writeups/Ceph_Multilaguage-RGW-Testing_JoannahNanjekye.md]: nit: Go sdk => Go SDK

**Comment by nanjekyejoannah-zz** [comment]
Requesting for reviews and merge if okay.

File: writeups/Ceph_Multilaguage-RGW-Testing_JoannahNanjekye.md [hunk 2]
```diff
@@ -1,68 +1,235 @@
...
 
-RGSoC was a huge success and a very memorable experience for me as I opened my first pull request then. The mentor was very welcoming and answered every question we asked him. I learned so much working on this project than even the experience I had from work. By and large I got to know that debugging is a very important skill when working on such big code bases, got to learn alot of python as I had been working with .Net and got to meet good people in the RGSoC community.
+If someone wanted to make a contribution and weren't be mentored or employed to do so
+they would need to be quite the self starter as well because seriously contributing to
+Ceph isn't something you can just do off and on.
 
-Through RGSoC I was also able to get a diversity grant to attend and speak at pyconZA, an opportunity that I have cherished todate because the conference opened my mind to many ideas. I gave a talk on [how to contribute to open source](https://speakerdeck.com/pyconzacontributing-to-python-open-source-infrastructure-and-projects-by-joannah-nanjekye/)
+Recommedation is you read carefully through the online documentation, watch videos
```
Comment (nikhita) [inline writeups/Ceph_Multilaguage-RGW-Testing_JoannahNanjekye.md]: My recommendation is that...

File: writeups/Ceph_Multilaguage-RGW-Testing_JoannahNanjekye.md [hunk 3]
```diff
@@ -1,68 +1,235 @@
...
+	## the "alt" user needs to have email set, too
+	email = john.doe@example.com
+	access_key = NOPQRSTUVWXYZABCDEFG
+	secret_key = nopqrstuvwxyzabcdefghijklmnabcdefghijklm
+
+Run the tests
+
+	S3TEST_CONF=your.conf ./virtualenv/bin/nosetests
+
+### Contribute! contribute! contribute!
```
Comment (nikhita) [inline writeups/Ceph_Multilaguage-RGW-Testing_JoannahNanjekye.md]: It's nicer to say Contribute! **C**ontribute! **C**ontribute! :wink:

File: writeups/Ceph_Multilaguage-RGW-Testing_JoannahNanjekye.md [hunk 4]
```diff
@@ -1,68 +1,235 @@
...
+## Getting Involved in RGW Testing
 
-#### About Ceph's Multilanguage RGW Testing project
+Ceph has many [projects](https://github.com/ceph). I will take you through the projects regarding RGW testing. To get involved, you have to first choose what test suite you want to contribute to and if you are not familiar with the programming language then you can go through some tutorials.
 
-I applied to ceph because of how interesting the project was. I anticipated I would build alot of capacity from working with Amazon Web services and working in three languages(Go, Java and C++) on three different test suites. My ambitious mind really wanted this. 
+### Prerequisites
 
-The project entails writing tests for aws using amazon Go, Java and C++ SDKS. RGW features an AWS s3 like interface against which tests are to be run. Ceph already had a python test suite that was testing RGW and my work was to extend the tests to the other three languages. 
+Most of the instructions in running these tests are based on the fact that you use linux. It is advisable that you get youself any linux distro like ubuntu or fedora but I would recommend fedora.
```
Comment (nikhita) [inline writeups/Ceph_Multilaguage-RGW-Testing_JoannahNanjekye.md]: s/youself/yourself

File: writeups/Ceph_Multilaguage-RGW-Testing_JoannahNanjekye.md [hunk 5]
```diff
@@ -1,68 +1,235 @@
...
+
+## Ping the community using these platforms
+
++ On IRC use #ceph-devel - Connect your client to: irc.oftc.net
++ Please send a mail to ceph-community@ceph.com if you would like an invite to the [Ceph slack channel](https://ceph-storage.slack.com/)
++ You can also use the mailing list ceph-devel@vger.kernel.org 
+
+## Happy Hacking!
+
+Ceph is a warm community. You may face some back and forth especially when building and running ceph but remember that it always works as @amaredia says. Feel free to reachout to me also (@Captain_Joannah on twitter) if you are stack. I can help out on especially the java and Go test suites.
```
Comment (nikhita) [inline writeups/Ceph_Multilaguage-RGW-Testing_JoannahNanjekye.md]: s/stack/stuck

**Review by nikhita — COMMENTED** [review]
Looks great!! :tada: 
Left a few nits.

**Comment by nikhita** [comment]
@nanjekyejoannah also, could you squash your commits to a single commit? :)

**Comment by nikhita** [comment]
@nanjekyejoannah why did you close this? :O

**Comment by nanjekyejoannah-zz** [comment]
Let me revert this ! Sorry

**Comment by nikhita** [comment]
No problem! :smile:

**Pull request closed without merging**